nixos/tests/systemd-machinectl.nix

```diff
@@ -59,8 +59,19 @@ import ./make-test-python.nix (
       machine.succeed("machinectl start ${containerName}");
       machine.wait_until_succeeds("systemctl -M ${containerName} is-active default.target");
 
+      # Test nss_mymachines without nscd
+      machine.succeed("LD_LIBRARY_PATH=\"/run/current-system/sw/lib\" getent -s hosts:mymachines hosts ${containerName}");
```

Comment:
```suggestion
      machine.succeed('LD_LIBRARY_PATH="/run/current-system/sw/lib" getent -s hosts:mymachines hosts ${containerName}');
```

Reply:
Why is the `LD_LIBRARY_PATH` needed here?

Reply:
The library directory contains the libnss_mymachines.so which getent shall use for host resolution. ... I applied your suggested change.